docs/source/integrations/cvat.rst

```diff
@@ -1462,6 +1465,66 @@ For example, let's upload some blurred images to CVAT for annotation:
    :alt: cvat-alt-media
    :align: center
 
+
+Linking the occluded property widget
```

Comment:
Feature request: when annotating existing fields with CVAT, if a boolean attribute called `occluded` exists with boolean values, use the `occlusion` widget by default.

I added the `recommend_attr_tool()` method for this reason